consensus/spos/bls/v2/subroundStartRound.go

```diff
@@ -224,18 +224,22 @@ func (sr *subroundStartRound) computeNumManagedKeysInConsensusGroup(pubKeys []st
 	numMultiKeysInConsensusGroup := 0
 	for _, pk := range pubKeys {
 		pkBytes := []byte(pk)
+
+		if sr.IsMultiKeyLeaderInCurrentRound() {
```

Comment:
this check here is introducing a sort of circular dependency
it cannot increment if it is not already incremented, due to the inner check of "backup machine needs to start acting as validator"